habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,310 @@
+                f"Not replacing data, generating symlink ({link_path}) and aborting download."
+            )
+            print("=======================================================")
+            # create a symlink to the versioned data
+            if link_path.exists():
+                os.unlink(link_path)
+            elif not link_path.parent.exists():
+                link_path.parent.mkdir(parents=True, exist_ok=True)
+            os.symlink(src=version_dir, dst=link_path, target_is_directory=True)
+            assert link_path.exists(), "Failed, no symlink generated."
+            return
+
+    # download new version
+    download_pre_args = data_sources[uid].get("download_pre_args", "")
+    download_post_args = data_sources[uid].get("download_post_args", "")
+
+    download_command = (
+        "wget --continue "
+        + download_pre_args
+        + data_sources[uid]["source"]
+        + " -P "
+        + data_path
+        + download_post_args
+    )
+    # print(download_command)
+    os.system(download_command)
+    assert os.path.exists(
+        os.path.join(data_path, data_sources[uid]["package_name"])
+    ), "Download failed, no package found."
+
+    # unpack
+    package_name = data_sources[uid]["package_name"]
+    if package_name.endswith(".zip"):
+        with zipfile.ZipFile(data_path + package_name, "r") as zip_ref:
+            zip_ref.extractall(version_dir)
+    else:
+        # TODO: support more compression types as necessary
+        print(f"Data unpack failed for {uid}. Unsupported filetype: {package_name}")
+        return
+
+    assert os.path.exists(version_dir), "Unpacking failed, no version directory."
+
+    # create a symlink to the new versioned data
+    if link_path.exists():
+        os.unlink(link_path)
+    elif not link_path.parent.exists():
+        link_path.parent.mkdir(parents=True, exist_ok=True)
+    os.symlink(src=version_dir, dst=link_path, target_is_directory=True)
+
+    assert link_path.exists(), "Unpacking failed, no symlink generated."
+
+    # clean-up
+    os.remove(data_path + package_name)
+
+    print("=======================================================")
+    print(f"Dataset ({uid}) successfully downloaded.")
+    print(f"Source: '{version_dir}'")
+    print(f"Symlink: '{link_path}'")
+    print("=======================================================")
+
+
+class AutoHelpParser(argparse.ArgumentParser):
+    def error(self, message):
+        sys.stderr.write("error: %s\n" % message)
+        self.print_help()
+        sys.exit(2)
+
+
+def main(args):
+    parser = AutoHelpParser()
+    arg_group1 = parser.add_mutually_exclusive_group()
+    arg_group1.add_argument(
+        "--uid",
+        type=str,
+        help="Unique ID of the data to download.",
+    )
+    arg_group1.add_argument(
+        "--list",
+        action="store_true",
+        help="List available datasource uid options and exit.",
+    )
+
+    parser.add_argument(
+        "--data_path",
```

Comment:
```suggestion
        "--data-path",
```
We've been using `-` for spacing in the rest of the codebase